tools/sigma/backends/datadog.py

```diff
@@ -0,0 +1,103 @@
+# Output backends for sigmac
+# Copyright 2021 Datadog, Inc.
+
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import re
+
+from sigma.backends.base import SingleTextQueryBackend
+from sigma.parser.condition import NodeSubexpression
+
+
+class DatadogLogsBackend(SingleTextQueryBackend):
+    """Converts Sigma rule into Datadog log search query."""
+
+    identifier = "datadog-logs"
+    active = True
+    config_required = False
+
+    andToken = " AND "
+    orToken = " OR "
+    notToken = "-"
+    subExpression = "(%s)"
+    listExpression = "(%s)"
+    listSeparator = " OR "
```

Comment:
Can we add a test for lists?

Reply:
Good call, I added such a test in 851e237.

Reply:
Thanks for adding the test https://github.com/DataDog/sigma/blob/a21fe1eb58243b4c328d4afe56f0cb94a9268698/tools/tests/test_backend_datadog.py#L107-L122

Are we positive on the choice of `" OR "` over `" AND "`?
 -  `" OR "` means that we search for any list that contains either one or the other
 -  `" AND "` means that we search for any list that contains at least both

Worth a comment and/or a note in the README?

Reply:
Yes, this is the expected behaviour defined in the [specification](https://github.com/SigmaHQ/sigma/wiki/Specification#lists).

Reply:
I added a comment about this in 63705cd.